### PR TITLE
Make changes to move from aos-ci to ci-pipeline as the repo name

### PR DESCRIPTION
**Pull request by arilivigni**
- Update macro and create repo-name: ci-pipeline
- replace aos-ci with ci-pipeline where macro can't apply

I will have to change the repo name once this is pushed.

### CODE REVIEW

**Comment by arilivigni** [comment]
[(5) Complete planning for all steps of the pipeline](https://trello.com/c/D80D9EcT/61-5-complete-planning-for-all-steps-of-the-pipeline)

**Comment by arilivigni** [comment]
[(3) [CI-Infrastructure] fedmsg format for publishing](https://trello.com/c/VtONedqu/67-3-ci-infrastructure-fedmsg-format-for-publishing)

File: jobs/rpmbuild.yaml [hunk 1]
```diff
@@ -18,10 +18,10 @@
             EOF
             cat task.env
             rsync -Hrlptv --stats -e ssh task.env rsync-password.sh builder@${{DUFFY_HOST}}:${{JOB_NAME}}
-            rsync -Hrlptv --stats --delete -e ssh aos-ci/ builder@${{DUFFY_HOST}}:${{JOB_NAME}}/aos-ci
+            rsync -Hrlptv --stats --delete -e ssh aos-ci/ builder@${{DUFFY_HOST}}:${{JOB_NAME}}/ci-pipeline
```
Comment (johnbieren) [inline jobs/rpmbuild.yaml]: Pretty sure the aos-ci in this line needs to be changed to ci-pipeline as well (right after the ssh)

File: jobs/ci-pipeline-defaults.yml [hunk 2]
```diff
@@ -34,8 +34,8 @@
           branches:
               - master
       - git:
-          url: "https://github.com/CentOS-PaaS-SIG/aos-ci"
-          basedir: aos-ci
+          url: "https://github.com/CentOS-PaaS-SIG/{repo-name}"
+          basedir: '{repo-name}'
```
Comment (johnbieren) [inline jobs/ci-pipeline-defaults.yml]: So do all the jobs need a repo-name: defined in them now?  I see it in rpmbuild_trigger.yaml, but not the others.  Not sure if I'm missing something.. haven't run this through jenkins-jobs test so not sure if it resolves

**Pull request merged**